components/vf-masthead/vf-masthead.scss

```diff
@@ -28,9 +59,19 @@ $vf-masthead__title-text--color: set-color(vf-color-white);
 
 
 .vf-masthead__heading {
-  @include set-type(heading--xl);
+  @include set-type(heading--xl, $custom-margin-bottom: 8px);
 
-  color: $vf-masthead__title-text--color;
+  color: inherit;
```

Comment:
We might be careful on changing this to inherit. The specificity inheritance seems to defer to the parent `.vf-masthead` over another stylesheet's closer `h1 { color: #222 }` ... but I'm sure that might vary from browser to browser or what might be going on elsewhere. 

or maybe here we could say `color: var(--vf-masthead__color--foreground)`

Reply:
Looking at how you're envisioning the theming options, I guess `color: var(--vf-masthead__color--foreground` wouldn't work. 

Perhaps instead we could only support "white" and "dark" foreground colours ... really that should be all someone should need? And if they need some crazy yellow text on blue background, they probably would need to be doing some other crazy stuff, like setting a bunch of other things to yellow and blue.

Reply:
Comment 1: I need to do some testing and thinking
Comment 2: Previously with this I’ve made use of JS to define the foreground colour so that it matches the correct contrast checks for WCAG … (so yes, either dark or light - but determined by the background colour at run time).

Reply:
yeah, let's not get hung up on this one ... it's something we can refine "in the future".